tests/rados/rados_prep.py

```diff
@@ -0,0 +1,320 @@
+import logging
+import re
+import time
+
+from ceph.ceph_admin import CephAdmin
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Prepares the cluster to run rados tests.
+    Actions Performed:
+    1. Create a Replicated and Erasure coded pools and write Objects into pools
+    2. Setup email alerts for sending errors/warnings on the cluster.
+        Verifies Bugs:
+        https://bugzilla.redhat.com/show_bug.cgi?id=1849894
+        https://bugzilla.redhat.com/show_bug.cgi?id=1878145
+    3. Enable logging into file and check file permissions
+        Verifies Bug : https://bugzilla.redhat.com/show_bug.cgi?id=1884469
+    Args:
+        ceph_cluster (ceph.ceph.Ceph): ceph cluster
+    """
+    log.info(run.__doc__)
+    config = kw.get("config")
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    config = kw.get("config")
+    unique_name = time.strftime("%M%S")
```

Comment:
can you think of using uuid here instead

Reply:
Made use of the uuid